src/BrowserInterop/Extensions/JsObjectWrapperBase.cs

```diff
@@ -6,8 +6,8 @@ namespace BrowserInterop.Extensions
 {
     public abstract class JsObjectWrapperBase : IAsyncDisposable
     {
-        internal JsRuntimeObjectRef JsObjectRef { get; private set; }
-        internal IJSRuntime JsRuntime { get; private set; }
+        protected internal JsRuntimeObjectRef JsObjectRef { get; private set; }
```

Comment:
we should remvoe the internal as we if you want to use it in 3rd party lib right ?

Reply:
@RemiBou It will be available by inheritance from `JsObjectWrapperBase` like that:

```csharp
class MyJsObject : JsObjectWrapperBase {
    void SomeMethod() {
        JsRuntime.Invoke ...
    }
}
```

Reply:
@RemiBou Yeah, maybe it is good idea to make it public at all ... In first commit I have made this properties public, but then decided to make it protected internal

Okay, I will change it to public back )

Reply:
I meant, remove the "internal" , only protected. "internal" is to allow access to only classes in the BrowserInterop assembly. protected alone is better

Reply:
@RemiBou ))) Okay )

Reply:
sorry you need it public as it's used elsewhere in the assembly :$

Reply:
> 
> 
> sorry you need it public as it's used elsewhere in the assembly :$

Yeah, I forget it (

That is why I have added protected internal modifier, in such way you can use it internally as well as clients can use it by inherit from JsWraperBase

Reply:
yes sorry again (https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/protected-internal) I thought it was protected AND internal, not protected OR internal. You can commit it, I'll try to deploy it today

Reply:
@RemiBou Done

Reply:
@RemiBou You can also `squash and merge` to reduce lots of commits where I add, remove and add back modifier )))))))))